### ISSUE TITLE
Cannot use my own pretrained model, Loss going NaN

### ISSUE THREAD

**Issue by Santzu-27** [OlafenwaMoses/ImageAI#836]
When i try to use my own pretrained model it runs a error: 
`trainer.setTrainConfig(object_names_array=["insetos"], batch_size=6, num_experiments=270, train_from_pretrained_model="yolov3_insetos_mAP-0.54737_epoch-30.pt")`

```
Generating anchor boxes for training images...
thr=0.25: 1.0000 best possible recall, 8.64 anchors past thr
n=9, img_size=416, metric_all=0.640/0.884-mean/best, past_thr=0.658-mean: 
pretrained weight loading failed. Defaulting to using random weight.
====================
Pretrained YOLOv3 model loaded to initialize weights
====================

```

But i "solved" the "pretrained weight loading failed. " error by putting the entire path way, but the results and losses keep giving to 0.0000 and Nan

`trainer.setTrainConfig(object_names_array=["insetos"], batch_size=6, num_experiments=270, train_from_pretrained_model="/home/oficinas40/2-BioIn/armadilhas/insetos/models/yolov3_insetos_mAP-0.54737_epoch-30.pt")

` 

```

Generating anchor boxes for training images...
thr=0.25: 1.0000 best possible recall, 8.64 anchors past thr
n=9, img_size=416, metric_all=0.641/0.884-mean/best, past_thr=0.659-mean: 
====================
Pretrained YOLOv3 model loaded to initialize weights
====================
Epoch 1/90
----------
Train: 
29it [02:03,  4.25s/it]
    box loss-> nan, object loss-> nan, class loss-> 0.00000
Validation:
31it [03:55,  7.60s/it]
    recall: 0.000000 precision: 0.000000 mAP@0.5: 0.000000, mAP@0.5-0.95: 0.000000
```

**Comment by YXIDA** [comment]
hello, finally the problem is solved no.
